Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -221,7 +227,7 @@ public final class AlertManager {
                     alertAt: nextTriggerDate,
                     title: notificationContent.title,
                     body: notificationContent.body,
-                    timeInterval: failureInterval,
+                    timeInterval: warningInterval,
```

Comment:
I worry about this change. If the `lastLoopDate` was not now, `timeUntilNotification` should be used, correct?

Reply:
Then the name should be updated to reflect this. At least to me it doesn't read this way.

Reply:
The Loop not looping warning has several different time intervals that it is triggered at, so a `warningInterval` seems like a descriptive name for that.  Or is it something else that you are wanting to make clearer?  Open to suggestions.

Reply:
I'm looking at `timeInterval` for the `StoredLoopNotRunningNotification`. I'm not sure how this is used, so it is hard to suggest an alternative. If this is not used, then just remove it. If it is needed, then its need will guide the name. `timeInterval` is too general and allows the reader to make false assumptions.

Reply:
It just keeps track of the particular time/warning interval of this notification, so it seems like it's accurately named to me?

Reply:
Removed this variable, which changes the serialized format of StoredLoopNotRunningNotification, which will potentially cause some historically issued alerts to not be recorded in the alert store is issued, affecting forensics, but not safety.